packages/Interface/src/DTK_Fortran_API.f90

```diff
@@ -111,6 +112,7 @@ subroutine swigc_DTK_set_function(farg1, farg2, farg3, farg4) &
    type(C_PTR), value :: farg4
   end subroutine
  end interface
+#endif
```

Comment:
@aprokop referred me to [this](https://stackoverflow.com/questions/18837456/how-to-use-doxygen-preprocessor-on-this-example-fortran-code).  Doxygen interprets the preprocessor directive like a comment because of the file extension.  I tried to use `EXTENSION_MAPPING` without much success but renaming the file `packages/Interface/src/DTK_Fortran_API.f90 -> packages/Interface/src/DTK_Fortran_API.F90`
did the trick.